styles.css

```diff
@@ -0,0 +1,71 @@
+p {
+    margin: 0;
+}
+
+.credit-card {
+    background-color: #425ecd;
+    font-size: large;
+    height: 240px;
+    width: 360px;
+    border-radius: 10px;
+    margin: 30px 0 0 20px;
+    display: flex;
+    flex-direction: column;
+}
+
+.credit-card-front > div {
+    margin-left: 40px;
+}
+
+.credit-card-front > p {
+    margin-left: 40px;
+}
+
+.credit-card-front > .number{
+    margin-top: 100px;
+    font-size: xx-large;
+}
+
+.credit-card-front > .information-wrapper {
+    display: inline-flex;
+    align-items: center;
+}
+
+.credit-card-front > .information-wrapper > p:first-child {
```

Comment:
You can simplify this to `.credit-card-front .information-wrapper p:first-child`

Reply:
done